Lykke.Job.LykkeJob/src/Lykke.Job.LykkeJob/RabbitPublishers/MyRabbitPublisher.cs

```diff
@@ -34,7 +34,6 @@ public void Start()
                 .SetPublishStrategy(new DefaultFanoutPublishStrategy(settings))
                 .PublishSynchronously()
                 .SetLogger(_log)
-                .SetConsole(new LogToConsole())
```

Comment:
Why?

Reply:
Using only _log is more flexible. It can be composite and contains LogToConsole() implementation thus you get the same functionality without two parallel logging system.

Reply:
Our devops guys asked to set console for rabbitMq sub\pub items everywhere - it helps them to monitor things better from console

Reply:
Also Console output via IConsole interface is different (more info) in comparison with ILog output

Reply:
Actually the whole implementation is an anti-pattern. We should use a container to register instance and here receive it as a dependency.

Reply:
For example:
https://github.com/LykkeCity/Lykke.RabbitMqDotNetBroker/blob/master/src/Lykke.RabbitMqBroker/Subscriber/RabbitMqSubscriber.cs#L154

Reply:
"Actually the whole implementation is an anti-pattern" - agree. I suggest to remove IConsole from RabbitMq classes completeley and replace it with standard logging.